### PR TITLE
Project 1: Initialize project with components

### PR DESCRIPTION
**Pull request by crgc**
### Features

- Uses `react-redux`
- Uses a "feature folder" approach and use the ducks pattern for the Redux files
- Components are reusable
- Added **React Router** and set up two `<Route>`s and `<Link>`s (Books and Categories)

### CODE REVIEW

File: src/pages/Books.js [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          <div className="book-info">
+            <div className="book-category">{book.category}</div>
+            <div className="book-title">{book.title}</div>
+            <div className="book-author">{book.author}</div>
+          </div>
+        </div>
+      ))}
+      <div className="gray-line" />
+    </div>
+  );
```
Comment (DeliceLydia) [inline src/pages/Books.js]: - [x] you need to add removing books the button wouldn't remove the books yet, this is just a placeholder for now

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+}
+
+export function loadBooks() {
+  return { type: LOAD };
+}
+
+export function createBook(book) {
+  return { type: CREATE, book };
+}
+
```
Comment (DeliceLydia) [inline src/redux/books/books.js:19]: - [x] You should render a form with inputs for the title and the author of the book
- [x] On the form add a button for submit the button wouldn't add the book *yet*, this is just a placeholder for now

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @crgc, 

Good job so far! However, there are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/pages/Books.js (hunk 1 above)
Comment (crgc) [reply src/pages/Books.js]: Done.

File: src/redux/books/books.js (hunk 2 above)
Comment (crgc) [reply src/redux/books/books.js:19]: Done.

File: src/pages/Books.js (hunk 1 above)
Comment (DeliceLydia) [reply src/pages/Books.js]: - [ ] I can't find the remove button still under the book as a placeholder please add that

File: src/redux/books/books.js (hunk 2 above)
Comment (DeliceLydia) [reply src/redux/books/books.js:19]: - [ ] Well addressed this but you can remember to add `author` inputs as well

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @crgc, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments I added under the previous review._


### Cheers and Happy coding!👏👏👏

Feel free to leave a question, or a comment on the PR thread if something is not clear 100%. 

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by crgc** [comment]
Ok - done. I've added everything you wanted. A remove button for each book and an input field for author.

**Review by DeliceLydia — APPROVED** [review]
# Congratulations 🎉  🎉 

## Hi @crgc, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard).. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**